Remotion/ObjectBinding/Web.Development.WebTesting/ControlObjects/BocCheckBoxControlObject.cs

```diff
@@ -75,6 +75,11 @@ public IReadOnlyList<string> GetValidationErrors ()
       return GetValidationErrors (GetScopeWithReferenceInformation());
     }
 
+    public IReadOnlyList<string> GetValidationErrorsForReadOnly ()
+    {
+      return GetValidationErrorsForReadOnly (GetScopeWithReferenceInformation());

```

Comment:
I saw that the test for this ControlObject failed, maybe `GetScopeWithReferenceInformation` is not the correct method?

Reply:
Related to https://github.com/re-motion/Trunk-SVN2Git/pull/35 ([RM-7305](https://re-motion.atlassian.net/browse/RM-7305)).